dotCMS/src/main/java/com/dotmarketing/filters/CookieServletResponse.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * ServletResponseCharacterEncoding.java
+ *
+ * Created on 8/10/2017
+ * @author Will Ezell
+ */
+package com.dotmarketing.filters;
+
+import static com.liferay.util.CookieUtil.COOKIES_HTTP_ONLY;
+
+import javax.servlet.ServletRequest;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.servlet.ServletRequest'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)